src/docker-compose.yml

```diff
@@ -28,9 +28,11 @@ services:
     stdin_open: true
     tty: true
     ports:
-      - "8080:8000"
+      - "8080:8080"
```

Comment:
This was a relic of when we were preparing these containers to run on Cloud.gov. Agree with the logic of having it match the message from Django.